src/WaferProber-Accretech_UFseries/main.py

```diff
@@ -196,25 +255,17 @@ def initialize(self):
                     "There is no wafer on the chuck, although 'Current wafer' is selected as Sweep value."
                 )
         else:
-            if self.prober.is_wafer_on_chuck():
-                raise Exception(
-                    "There is already a wafer on the chuck. Please terminate the ongoing lot process or "
-                    "remove the wafer.")
-
-            if self.sweep_value_die == "Current die":
-                raise Exception(
-                    "Performing a wafer variation for the current die is not possible as the current die is"
-                    "not defined when no wafer is on the chuck.")
-
             if self.sweep_value_die == "Current die":
                 raise Exception(
                     "Performing a wafer variation for the current die is not possible as the current die is"
-                    "not defined when no wafer is on the chuck.")
+                    "not defined for a wafer variation.")
+            
+            if self.prober.is_wafer_on_chuck():
+                self.prober.unload_all_wafers()
 
-            cassette_status = self.prober.request_cassette_status()
-            if cassette_status == "000000":
-                self.prober.sense_wafers()  # Wafer sensing "jw" command
+            self.check_and_sense_wafers()
 
+            """
```

Comment:
Is this code obsolete? Then it can be removed.
Or is it needed when users have some special requirements and need to re-enable it? Then there should be an explanation when it is needed and how it should be used

Reply:
I removed it